shell/packages/sandstorm-db/db.js

```diff
@@ -153,6 +153,7 @@ Packages = new Mongo.Collection("packages");
 //   url:  When status is "download", the URL from which the SPK can be obtained, if provided.
 //   isAutoUpdated: This package was downloaded as part of an auto-update. We shouldn't clean it up
 //     even if it has no users.
+//   isPreinstalled: This package was downloaded in order to be pre-installed by users.
```

Comment:
Actually I believe the check should only be "Is this packageId marked for pre-install". I purposely denormalized packageId into the setting for situations like this, so that it's more feasible to implement non-app store apps as preinstalled (someday).